portal/server/controllers/dashboard/facilities/facilities-filters-query.js

```diff
@@ -16,17 +16,29 @@ const dashboardFacilitiesFiltersQuery = (
   user,
 ) => {
   const query = {};
+  let dashboardFilters = filters;
+
   // if user is admin, then deal.bank.id should not be set so cannot see all
   if (!isSuperUser(user)) {
     query.$and = [
       { 'deal.bank.id': user.bank.id },
     ];
   }
 
-  if (filters.length) {
+  const filtered = [];
+  // removes _crsf from facilitiesFilter
+  Object.values(dashboardFilters).forEach((value) => {
```

Comment:
We can simplify this using `filter` method

```
Object.values(dashboardFilters)
  .filter((v) => !v._csrf)
  .map((v) => filtered.push(v));

Reply:
done